### ISSUE TITLE
VisLeafletFlowMap: 'layer.getMapLibreMap is not a function' when using renderer="raster"

### ISSUE THREAD

**Issue by ereid7** [f5/unovis#131]
VisLeafletFlowMap: 'layer.getMapLibreMap is not a function' when using renderer="raster"
This only seems to happen when the page first loads:

```
Error in /turbo_modules/@unovis/ts@1.0.2/components/leaflet-map/index.js (172:39)
layer.getMaplibreMap is not a function
```

Using Firefox on Mac

See stackblitz:
https://stackblitz.com/edit/react-ts-ht4tcq?file=App.tsx

**Comment by rokotyan** [comment]
@ereid7 Thanks for the stackblitz! 

@reb-dev Can you please look into it?

**Comment by rokotyan** [comment]
@ereid7 Fixed in version `1.0.3-beta.6`
